dev/com.ibm.ws.jca_fat_example_anno/fat/src/test/examplera/anno/ResourceAdapterExampleTest.java

```diff
@@ -74,7 +76,7 @@ public void testAddAndFind() throws Exception {
 
         // find
         runTest("?functionName=FIND&capital=Saint%20Paul",
-                "Successfully performed FIND with output: {area=86939, capital=Saint Paul, population=5379139, state=Minnesota}");
+                "Successfully performed FIND with output: [area=86939, capital=Saint Paul, population=5379139, state=Minnesota]");;
```

Comment:
This looks like a change in behavior. What caused the output to change from `{}` to `[]`? 
Also, this line has two `;`s at the end.

Reply:
Will fix the ;; good catch